src/csharp/Intel/Iced/Intel/Extended/ExtendedRegisters.cs

```diff
@@ -0,0 +1,10 @@
+namespace Iced.Intel
+{
+	public static partial class ExtendedRegisters {
+		public static readonly ExtendedMemoryOperandFactory __ = new ExtendedMemoryOperandFactory(MemoryOperandSize.None);
+		public static readonly ExtendedMemoryOperandFactory __byte_ptr = new ExtendedMemoryOperandFactory(MemoryOperandSize.BytePtr);
+		public static readonly ExtendedMemoryOperandFactory __word_ptr = new ExtendedMemoryOperandFactory(MemoryOperandSize.WordPtr);
+		public static readonly ExtendedMemoryOperandFactory __dword_ptr = new ExtendedMemoryOperandFactory(MemoryOperandSize.DwordPtr);
+		public static readonly ExtendedMemoryOperandFactory __qword_ptr = new ExtendedMemoryOperandFactory(MemoryOperandSize.QwordPtr);
+	}
```

Comment:
May be better set names as 
 * `__` -> `ptr` e.g. `ptr[rax+8]`
 * `__byte_ptr` -> `ptr_byte` e.g. `ptr_byte[rax]`
and etc.

Reply:
ptr is a bit too annoying because it is too generic and when imported via `using static` it would easily pollute user variable name space, unlike other registers (like `rax`), that's why I made the choice for `__`... or we could use `__ptr` in that case. Note that the case of using `__` is only relevant when MemoryOperand addressing is determined by the register (like `mov rm32, r32`), in that case you don't need to specify that you want a `__dword_ptr`.

for `byte_ptr` I could remove the leading `__` but found that it was easier for discoverability and consistency (along `__`) when creating a memory operand. The order `byte ptr` is also important (not `ptr byte`) because of assemblers are usually using the `byte ptr` naming order instead.